lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -6,19 +6,22 @@
 This class handles the sharing of secrets between sharded components, adding shards, and removing
 shards.
 """
+import json
 import logging
 
+from charms.mongodb.v1.helpers import add_args_to_env, get_mongos_args
+from charms.mongodb.v1.mongos import MongosConnection
 from ops.charm import CharmBase, EventBase
 from ops.framework import Object
-from ops.model import WaitingStatus
+from ops.model import ActiveStatus, MaintenanceStatus, WaitingStatus
 
 from config import Config
 
 logger = logging.getLogger(__name__)
 KEYFILE_KEY = "key-file"
 KEY_FILE = "keyFile"
 HOSTS_KEY = "host"
-CONFIG_SERVER_URI_KEY = "config-server-uri"
+CONFIG_SERVER_DB_KEY = "config-server-db"
 
 # The unique Charmhub library identifier, never change it
 LIBID = "58ad1ccca4974932ba22b97781b9b2a0"
```

Comment:
bump libpatch

Reply:
hmm, is there should be change in libpatch version?

Reply:
yes, whenever code changes the libpatch must be bumped